webdriver_wharf/__init__.py

```diff
@@ -1,6 +1,21 @@
 import logging
 from threading import RLock
 
+import docker
+
+try:
+    docker.AutoVersionClient
+except AttributeError:
+    pass  # yay no docker_py
+else:
+    raise ImportError("""\
+docker-py installation detect
+
+this breaks the good docker client
```

Comment:
docker-py detected!
This is incompatible with the newer docker package.
Please uninstall both clients and then reinstall the newer
docker package to run Wharf.